etcddb/etcddb_server.go

```diff
@@ -76,10 +77,12 @@ func getEtcdConf(conf *PaymentChannelStorageServerConf, cluster string) *embed.C
 		Host:   fmt.Sprintf("%s:%d", conf.Host, conf.PeerPort),
 	}
 
-	fmt.Printf("name: '%s'\n", conf.ID)
-	fmt.Printf("client url: '%v'\n", clientURL)
-	fmt.Printf("peer   url: '%v'\n", peerURL)
-	fmt.Printf("initial cluster: '%s'\n", cluster)
+	log.WithFields(log.Fields{
+		"etcd id":         conf.ID,
+		"client url":      clientURL,
+		"peer   url":      peerURL,
+		"initial cluster": cluster,
+	}).Debug()
```

Comment:
I would use Info() it is very useful to know which configuration values are actually used.